common/src/main/java/com/wynntils/wynn/model/map/poi/WaypointPoi.java

```diff
@@ -7,8 +7,16 @@
 import com.wynntils.gui.render.Texture;
 
 public class WaypointPoi extends IconPoi {
+
+    private final PointerPoi pointer;
+
```

Comment:
feels a bit weird on first look but also clever in a way. should be fine to keep this

Reply:
I don't see a better alternative for now